themes/digital.gov/src/scss/new/guides/_guide-content.scss

```diff
@@ -1,35 +1,48 @@
 @use "uswds-core" as *;
 
-.dg-guide__content-body {
-  margin-bottom: units(5);
-  overflow: hidden;
-
-  p {
-    max-width: none;
-  }
+.dg-guide__container {
+  display: flex;
 }
 
-.dg-guide__content-header {
-  margin: units(5) 0;
+.dg-guide__content {
```

Comment:
Styling content on a per-page basis is something we should get away from.

Styles should be consistently applied throughout the site.

Can we create a card to follow-up and address this?

Reply:
Good idea! Added to [this existing card](https://trello.com/c/u2q0NNah/2397-dg-create-global-styles-partial) about global styles.

Reply:
Thanks for updating that! Resolving thread.